### DISCUSSION TITLE
Sponsorship

### DISCUSSION THREAD

**Discussion by ben519** [panva/jose#664] [Q&A]
Hey, I use `jose` for my platform [Scipress.io](https://www.scipress.io/). I was hoping I could sponsor this project in exchange for advertising my company on your README. I want to advertise my site, but I also want to say thanks for maintaining the package!

Let me know if this is possible and what it would entail. Thanks!

**Comment by panva** [comment]
You're most certainly welcome to sponsor as a token of appreciation for the value my projects provide you. I would be super happy.

As far as advert space goes, other than years old agreements on two of my repos and I don't do logo placements anymore for new projects.

**Reply by ben519** [reply]
Fair enough. Thanks for the prompt reply!
